packages/integrations/markdoc/test/fixtures/render-with-indented-components/src/content/blog/with-indented-components.mdoc

```diff
@@ -0,0 +1,24 @@
+---
+title: Post with indented components
+---
+
+## Post with indented components
+
+This uses a custom marquee component with a shortcode:
+
+{% marquee-element direction="right" %}
+	I'm a marquee too!
+
+	{% marquee-element direction="right" %}
+		I'm an indented marquee!
+
+		### I am an h3!
+	{% /marquee-element %}
+
+	And a nested code block:
+
+	```js
+	const isRenderedWithShiki = true;
+	```
```

Comment:
This could be a nitpick, but I haven't seen anything about it in the documentation. Does the indentation work with tabs and spaces regardless? If so, I believe we should mention it. For example, this code block has **tabs**, but the documentation has **spaces** (which are forced to use in markdown documents)

Reply:
I didn't consider tabs v spaces. I have tested this now and it works with either. I've ave added a comment at the top of the code example